classes/author.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'item'
+
+class Author
+  attr_accessor :first_name, :last_name, :items
+
+  attr_reader :id
+
+  def initialize(f_name, l_name)
+    @id = Random.rand(1..1000)
+
+    @first_name = f_name
+
+    @last_name = l_name
+
+    @items = []
+  end
+
+  def add_item(new_item)
+    @items << (new_item)
+  end
```

Comment:
- [ ] I am unable to add an author successfully, the program takes in input and upon enering the last value, loops back to the main menu, kindly fix this issue

Reply:
Sally is a good girl